tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -772,6 +772,14 @@ platform_tests/mellanox:
     conditions:
       - "asic_type not in ['mellanox', 'nvidia-bluefield']"
 
+platform_tests/mellanox/test_check_sfp_eeprom.py:
+  skip:
+    reason: "202405 and 202411 not support command like 'mlxlink -d /dev/mst/mt53104_pci_cr0 -p 1 -m' on mellanox spc3 devices when software control is enabled"
+    conditions_logical_operator: or
+    conditions:
+      - "SN4 in hwsku"
```

Comment:
Could you also add releases to skip?

Something like this:  "release in ['202405', '202411']"

Reply:
will do 
thanks

Reply:
@prabhataravind , Please review. Thanks